react/features/settings/components/web/ModeratorTab.tsx

```diff
@@ -182,10 +206,17 @@ class ModeratorTab extends AbstractDialogTab<IProps, any> {
                 <Checkbox
                     checked = { followMeEnabled && !followMeActive }
                     className = { classes.checkbox }
-                    disabled = { followMeActive }
+                    disabled = { followMeActive || followMeRecorderActive }
                     label = { t('settings.followMe') }
                     name = 'follow-me'
                     onChange = { this._onFollowMeEnabledChanged } />
+                <Checkbox
+                    checked = { followMeRecorderEnabled && !followMeRecorderActive }
+                    className = { classes.checkbox }
+                    disabled = { followMeRecorderActive || followMeActive }
+                    label = { t('settings.followMeRecorder') }
+                    name = 'follow-me'
```

Comment:
shouldn't we put different value in name?

Reply:
fixed.